ci/ray_ci/tester.py

```diff
@@ -154,7 +161,9 @@ def main(
     if not bazel_workspace_dir:
         raise Exception("Please use `bazelisk run //ci/ray_ci`")
     os.chdir(bazel_workspace_dir)
-    docker_login(_DOCKER_ECR_REPO.split("/")[0])
+    # TODO: remove the if condition when wanda is supported for windows
+    if operating_system == "linux":
+        docker_login(_DOCKER_ECR_REPO.split("/")[0])
```

Comment:
this has not much to do with wanda, this is just AWS ec2 machine credentials.

wanda is the container image building tool.